### PR TITLE
proposal: tc plugin for netqos

### PR DESCRIPTION
**Pull request by lucming**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Comment by zwzhang0107** [comment]
origin PR：#1954

File: docs/proposals/koordlet/20231208-support-netqos.md [hunk 1]
```diff
@@ -330,7 +330,59 @@ After that, the netqos plugin will implement the network limiting operation base
...
+      ipset create ls_class hash:net
+      iptables -t mangle -A POSTROUTING -m set --match-set ls_class src  -j CLASSIFY --set-class 1:3
+      ipset create be_class hash:net
+      iptables -t mangle -A POSTROUTING -m set --match-set be_class src  -j CLASSIFY --set-class 1:4
+      ```
+    - For the pod of host network.
+      The speed limit can be achieved directly through the `net_cls` cgroup(Because there is no network namespace switching, the packet classid marking will not be lost, so you can enter the specific class, thus achieving the speed limit).
+
+    **todo:**  
+    Limits the network bandwidth in the input direction.
```
Comment (zwzhang0107) [inline docs/proposals/koordlet/20231208-support-netqos.md:385]: ingress direction

Comment (lucming) [reply docs/proposals/koordlet/20231208-support-netqos.md]: done

**Review by hormes — APPROVED** [review]

**Pull request merged**